.github/actions/spelling/allow/colors.txt

```diff
@@ -0,0 +1,117 @@
+alice
```

Comment:
This contains the colors that were valid -- numbers (`0`..`9`) and `-` are treated as word breaks by check-spelling.